tests/end2end-tests/data_gen/process_movielens.py

```diff
@@ -139,4 +147,46 @@ def write_data_parquet(data, data_file):
 # generate synthetic user data with label
 user_labels = np.random.randint(11, size=feat.shape[0])
 user_data = {'id': user['id'].values, 'feat': feat, 'occupation': user['occupation'], 'label': user_labels}
-write_data_parquet(user_data, '/data/ml-100k/users_with_synthetic_labels.parquet')
\ No newline at end of file
+write_data_parquet(user_data, '/data/ml-100k/users_with_synthetic_labels.parquet')
+
+
+user_feat_only = {'feat': feat}
```

Comment:
Are the codes below for dist partition test?
Why we store features and labels separately here?

Reply:
Yes, we need to store them separately as that's the expected input format for the DistPart pipeline: https://docs.dgl.ai/guide/distributed-preprocessing.html#chunked-graph-format